demos/llm_npu/README.md

```diff
@@ -203,7 +201,6 @@ client = OpenAI(
 )
 
 response = client.chat.completions.create(
-    model="mistralai/Mistral-7B-Instruct-v0.2",
```

Comment:
test fails with error:
TypeError: Missing required arguments; Expected either ('messages' and 'model') or ('messages', 'model' and 'stream') arguments to be given

Reply:
model=meta-llama/Llama-3.1-8B-Instruct